### ISSUE TITLE
Miktex failures in CI

### ISSUE THREAD

**Issue by tvercaut** [MIDL-Conference/MIDLLatexTemplate#23]
As shown here
https://github.com/MIDL-Conference/MIDLLatexTemplate/actions/runs/3509663062/jobs/5879019159
the windows CI is failing. It also looks like it's relying on an ancient version of miktex.

A first step could be to try and update miktex for CI purposes.

**Comment by tvercaut** [comment]
Seems others are facing the same issue:
https://github.com/actions/runner-images/issues/6712
https://github.com/MiKTeX/miktex/issues/1240

For the record, asking to use a `windows-2019` instead of `windows-latest` (i.e.`windows-2022`) runner doesn't help.

**Comment by HKervadec** [comment]
I had investigated it a bit when it first happened, I think it was the download URL of miktex that had changed. I had found the new one, but ran into another issue; was waiting to get back on a windows machine to test the installer first. 

As the failing CI failed after 528e88893479e720e4304dc1d513e7cd589be15a I had figured it was mostly a CI issue, not the latex itself. But as the MIDL deadline is looming I should fix that asap. 

For now I merged your pull request (as you've also updated the macos and linux versions, thanks!), and will continue investigating for the windows build.
